### PR TITLE
Ali/fix event sending disabled feature

### PR DESCRIPTION
**Pull request by mikeng13**

### CODE REVIEW

**Comment by mikeng13** [comment]
build

File: spec/project_spec.rb [hunk 1]
```diff
@@ -834,7 +834,8 @@ class InvalidErrorHandler; end
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Feature 'multi_variate_feature' is enabled for user 'test_user'.")
     end
 
-    it 'should return false, if the user is bucketed into a feature experiment but the featureEnabled property is false' do
+    it 'should return false and send impression if the user is bucketed into a feature experiment but the featureEnabled property is false' do
+      allow(project_instance.event_dispatcher).to receive(:dispatch_event).with(instance_of(Optimizely::Event))
```
Comment (aliabbasrizvi) [inline spec/project_spec.rb]: Can we look at content of dispatch event itself?

Comment (mikeng13) [reply spec/project_spec.rb]: I did try and it was giving me a lot of headache lol. The compat suite already checks the payload  so I'd say we merge this in!

Comment (aliabbasrizvi) [reply spec/project_spec.rb]: Go for it.

**Pull request merged**